lib/web_of_science/query_author.rb

```diff
@@ -1,88 +1,29 @@
 # frozen_string_literal: true
 
 module WebOfScience
-  # Use author name-institution logic to find WOS publications for an Author
+  # Search WoS for all publications for a given author, using both the name and ORCID queries, and then de-duping the UIDs
+  # fetch the wos uids for the given author
+  # e.g. WebOfScience::QueryAuthor.new(author).uids
   class QueryAuthor
+    attr_reader :orcid_query, :name_query
+
     def initialize(author, options = {})
       raise(ArgumentError, 'author must be an Author') unless author.is_a? Author
 
-      @identities = [author].concat(author.author_identities.to_a) # query for alternates once, not multiple times
-      @options = options
+      @orcid_query = QueryOrcid.new(author, options)
+      @name_query = QueryName.new(author, options)
     end
 
-    # Find all WOS-UIDs for an author
+    # Find all WOS-UIDs for an author using both ORCID and Name queries, and then de-dupe
     # @return [Array<String>] WosUIDs
     # Implementation note: these records have a relatively small memory footprint, just UIDs
     def uids
-      queries.search(author_query).merged_uids
+      (orcid_query.uids + name_query.uids).uniq
     end
```

Comment:
we now join the results from the name and orcid queries and unique to prevent dupes